### DISCUSSION TITLE
Published ARCs preview

### DISCUSSION THREAD

**Discussion by Mestizia** [nfdi4plants/.github#3] [Ideas]
This post is part of a series of suggestions endorsed by multiple data stewards involved daily in dataPLANT. Here, we would like to propose changes to improve the ARC publication routine, particularly focusing on the clarity and user-friendliness of published ARCs. Below are our suggestions for enhancing the ARC visualization and publication process, including a clearer summary display and a more accessible way of interacting with ARCs.

***1. Reintroduce arcsummary.md:***
Currently, the visualization of published ARCs primarily shows the arc.json file, which is not very intuitive for reviewers or users unfamiliar with the specific JSON format. For example, the preview available at this link is not user-friendly and may confuse a reviewer. By bringing back the arcsummary.md file, we can provide a more readable and accessible overview of the ARC's content. This summary should include an abstract and possibly one figure to make the content more appealing and informative. Additionally, we suggest creating guidelines to help users format their summaries effectively.
![](https://pad.hhu.de/uploads/99fcda85-9cad-46cf-825d-26b58cca30d9.png)

***2. Incorporate a README.md:***
Automatic creation of a README.md file that contains the basics of ARCs can serve as a helpful guide for users, outlining the contents of the ARC, explaining the structure, and offering instructions on how to use or interpret the data. This would enhance the user experience by providing a quick reference point and making the archives more approachable, especially for those new to the platform.

***3. Enhance ARC DOI Representation:***
When an ARC is assigned a DOI, the corresponding link currently resolves to the archive platform and presents a JSON file. Instead, we propose that the DOI should resolve to a more user-friendly view, such as the arcsummary.md file or README.md . This change would improve readability and accessibility, ensuring that the essential information is immediately visible and easy to understand. 

Specifically, these changes aim at improving the readability at first glance of published ARCs by third parties not familiar with dataPLANT such as scientific reviewers.

***Summary***

These changes aim to make ARCs more accessible and user-friendly, enhancing the overall experience for both creators and reviewers. By reintroducing the arcsummary.md, adding a README.md, and improving DOI resolution to more readable formats, we believe the ARC publication routine can significantly improve. 

We welcome any feedback or further discussion on these points to refine our approach. Thank you for considering these suggestions!

**Comment by Mestizia** [comment]
@Brilator @SabrinaZander @eik-dahms @angelaKranz

**Comment by kMutagene** [comment]
> 1. Reintroduce arcsummary.md:
Currently, the visualization of published ARCs primarily shows the arc.json file, which is not very intuitive for reviewers or users unfamiliar with the specific JSON format

@j-bauer @Zerskk i think this should be easy, as the summary markdown is now contained in the package with the exported json again IIRC?

> 2. Incorporate a README.md:
Automatic creation of a README.md file that contains the basics of ARCs can serve as a helpful guide for users, outlining the contents of the ARC, explaining the structure, and offering instructions on how to use or interpret the data. This would enhance the user experience by providing a quick reference point and making the archives more approachable, especially for those new to the platform.

Any automated commits to arcs have the potential to disrupt users, as local and remote ARCs can get out-of-sync. I think an approach via a bot that creates a pull request that can be accepted by the user would be a good approach. This can also be adapted for any kind of automated things done to ARCs, e.g. suggesting ontologies etc.

**Comment by Brilator** [comment]
IMO this is solved with new ARChigator, ARCWebView plugin, etc.
Close? @kMutagene
